libminifi/include/core/state/UpdatePolicy.h

```diff
@@ -69,11 +69,9 @@ class UpdatePolicy {
   }
 
  protected:
-  explicit UpdatePolicy(const UpdatePolicy &other)
-      : enable_all_(other.enable_all_), properties_(other.properties_) {
-  }
+  UpdatePolicy(const UpdatePolicy &other) = default;
 
-  explicit UpdatePolicy(const UpdatePolicy &&other)
+  UpdatePolicy(UpdatePolicy &&other)
```

Comment:
The move ctor could be defaulted, too.

Reply:
Changed the move constructor and the move assignment as well.